plugin.video.amazon-test/resources/language/resource.language.pl_pl/strings.po

```diff
@@ -274,125 +270,123 @@ msgstr "Ścieżka"
 
 msgctxt "#30063"
 msgid "Create Video Information File"
-msgstr ""
+msgstr "Utwórz plik informacji"
 
 msgctxt "#30064"
 msgid "Mediasource Movie Title"
-msgstr ""
+msgstr "Tytuł filmu z źródła"
 
 msgctxt "#30065"
 msgid "Mediasource TV Title"
-msgstr ""
-
-msgctxt "#30066"
-msgid "Send video progress to Amazon at least at percent played (0 to disable)"
-msgstr ""
+msgstr "Tytuł serialu z źródła"
 
 msgctxt "#30070"
 msgid "Miscellaneous"
-msgstr ""
+msgstr "Różne"
 
 msgctxt "#30071"
 msgid "Display TV-Show instead of Seasons in Watchlist"
-msgstr ""
+msgstr "Pokazuj serial zamiast sezonów w obserwowanych"
 
 msgctxt "#30072"
 msgid "Verbose Logging"
-msgstr ""
+msgstr "Pełne rejestrowanie"
 
 msgctxt "#30073"
 msgid "Display Paid Content"
-msgstr ""
+msgstr "Wyświetl płatną zawartość"
 
 msgctxt "#30074"
 msgid "Remote Control support"
-msgstr ""
+msgstr "Obsługa zdlanego sterowania"
 
 msgctxt "#30075"
 msgid "UP/DOWN controls Kodi Volume"
-msgstr ""
+msgstr "GÓRA/DÓŁ steruje głośnością Kodi"
 
 msgctxt "#30076"
 msgid "Watchlist/Library Order"
-msgstr ""
+msgstr "Kolejność Obserwowanych/Biblioteki"
 
 msgctxt "#30077"
 msgid "Recently Added"
-msgstr ""
+msgstr "Ostatnio dodane"
 
 msgctxt "#30078"
 msgid "Title Ascending"
-msgstr ""
+msgstr "Tytuł rosnąco"
 
 msgctxt "#30079"
 msgid "Title Descending"
-msgstr ""
+msgstr "Tytuł malejąco"
 
 msgctxt "#30080"
 msgid "Automatically Watchlist export"
-msgstr ""
+msgstr "Automatycznie eksportuj obserwowane"
 
 msgctxt "#30081"
 msgid "Intro/Recap scenes processing"
-msgstr ""
+msgstr "Przetwarzanie czołówki/podsumowania"
 
 msgctxt "#30082"
 msgid "Do nothing"
-msgstr ""
+msgstr "Nic nie rób"
 
 msgctxt "#30083"
 msgid "Show Skip Button"
-msgstr ""
+msgstr "Pokaż przycisk pominięcia"
 
 msgctxt "#30084"
 msgid "Autoskip Intro"
-msgstr ""
+msgstr "Pomijaj czołówkę automatycznie"
 
 msgctxt "#30085"
 msgid "Autoskip Recap"
-msgstr ""
+msgstr "Pomijaj podsumowanie automatycznie"
 
 msgctxt "#30086"
 msgid "Autoskip All"
-msgstr ""
+msgstr "Pomijaj wszystko automatycznie"
 
 msgctxt "#30087"
 msgid "Clear cache"
-msgstr ""
+msgstr "Wyczyść cache"
 
 msgctxt "#30088"
 msgid ""
 "Clear cached catalog or video data?\n"
 "Video data includes series/season metadata."
 msgstr ""
+"Wyczyścić katalog lub dane wideo w pamięci podręcznej?\n"
+"Dane wideo obejmują metadane serialu/sezonu."
 
 msgctxt "#30089"
 msgid "No"
-msgstr ""
+msgstr "Nie"
 
 msgctxt "#30090"
 msgid "Catalog"
-msgstr ""
+msgstr "Katalog"
 
 msgctxt "#30091"
 msgid "Video data"
-msgstr ""
+msgstr "Dane wideo"
 
 msgctxt "#30092"
 msgid "Both"
-msgstr ""
+msgstr "Oba"
 
 msgctxt "#30100"
 msgid "Library"
 msgstr "Biblioteka"
 
 msgctxt "#30101"
-msgid "Toggle Watched State"
+msgid "Przełącz stan oglądania"
 msgstr ""
```

Comment:
You replaced the source instead of inserting the target, this string won't work.

Reply:
Ok - my fault